### PR TITLE
與後端連接

### PR DESCRIPTION
**Pull request by quan0715**
@Bryant-Tang 我有動到你的東西，你再看一下程式碼原本是List<String> 我改成 List<dynamic> 不然過不了

### CODE REVIEW

**Review by Dilamissu — DISMISSED** [review]

File: lib/model/profile_model.dart [hunk 1]
```diff
@@ -58,7 +64,7 @@ class ProfileModel extends DataModel<ProfileModel> {
 
   /// convert two `List<String>` to `List<ProfileTag>`
   List<ProfileTag> _fromFirestoreTags(
-      List<String> tagList, List<String> tagContentList) {
+      List<dynamic> tagList, List<dynamic> tagContentList) {
```
Comment (Bryant-Tang) [inline lib/model/profile_model.dart]: line 67 should be :
      List<String> tagList, List<String> tagContentList) {

and change line 110 to :
            ? _fromFirestoreTags(data['tags'] as List<String>, data['tag_contents'] as List<String>)

**Review by Bryant-Tang — APPROVED** [review]

File: lib/model/profile_model.dart (hunk 1 above)
Comment (Bryant-Tang) [reply lib/model/profile_model.dart]: resolve
with direct commit (!!?

**Review by Dilamissu — DISMISSED** [review]

File: lib/model/profile_model.dart (hunk 1 above)
Comment (quan0715) [reply lib/model/profile_model.dart]: Get it, It is being fixed, and commit again.

**Comment by quan0715** [comment]
@Dilamissu @Bryant-Tang @S-Garry  重新 push 了一版你們在幫我approve 一下

**Review by S-Garry — APPROVED** [review]

**Review by Bryant-Tang — APPROVED** [review]

**Review by S-Garry — APPROVED** [review]

**Review by Dilamissu — APPROVED** [review]

**Pull request merged**